.github/workflows/publish_plugins.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish Built-in Plugins
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish plugins
+    runs-on: ubuntu-latest
+    env:
+      SCARB_REGISTRY_AUTH_TOKEN: ${{ secrets.SCARB_REGISTRY_AUTH_TOKEN }}
```

Comment:
What token do you think we should keep in our GitHub secrets? Do we want to use some sort of an organization account, or will my personal token do for now? :D

Also, do you think we need any expiration date for it?